pulsar-metadata/src/main/java/org/apache/pulsar/metadata/impl/AbstractMetadataStore.java

```diff
@@ -136,6 +139,7 @@ public CompletableFuture<Boolean> asyncReload(String key, Boolean oldValue,
                         }
                     }
                 });
+        CacheMetricsCollector.CAFFEINE.addCache(metadataStoreName + "-exists", childrenCache);
```

Comment:
this should use `existsCache`, will be fix in https://github.com/apache/pulsar/pull/24407